id/iema/2016/main.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by yazan on 1/12/17.
+ */
+
+//scroll to top button
+$( document ).ready(function() {
+    $('.go-top').click(function() {
+        $('body,html').animate({
+            scrollTop: 0
+        }, 'slow');
+        return false;
+    });
+});
+
+//smooth scroll animated
+var nav = $('#header')
+nav.find('a').on('click', function () {
+    var $el = $(this)
+        , id = $el.attr('href');
+
+    $('html, body').animate({
+        scrollTop: $(id).offset().top
+    }, 500);
+
+    return false;
+});
+
+//fixed sticky navigation menu once start scroll
+$(document).ready(function() {
+
+    $(window).bind('scroll', function() {
+
+        if($(window).scrollTop() > 171){
```

Comment:
Put 171, in a variable so that we can easily change it in future,
`heightScrollTop = 171`